manifests/cf-manifest/manifest/000-base-cf-deployment.yml

```diff
@@ -87,7 +87,7 @@ releases:
 stemcells:
   - alias: default
     name: bosh-aws-xen-hvm-ubuntu-trusty-go_agent
-    version: "3445.16"
+    version: "3468"
```

Comment:
Is there a reason to use this version instead of `3468.1` or `3468.5`?

Given the timelines of [the stemcell releases](https://github.com/cloudfoundry/bosh-linux-stemcell-builder/releases) and the recent crop of USNs that have been fixed, I worry that this version won't include the fixes for the USNs.

In general, when doing CF upgrades, we use the latest version from the recommended stemcell series.

Reply:
I'm not certain if they hadn't been published on bosh.io at the time (no timestamps on the releases) or were missed on the upgrade story. PR to update is located at https://github.com/alphagov/paas-cf/pull/1101